keyboards/kbd8x/kbd8x.c

```diff
@@ -39,5 +39,26 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
 void led_set_kb(uint8_t usb_led) {
 	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
 
+  if(usb_led & (1<<USB_LED_NUM_LOCK))
```

Comment:
It might be cleaner to write this as a terniary statement

```c
(usb_led & (1 << USB_LED_NUM_LOCK) ? num_led_on() : num_led_off();
```

along with the others in this file.

Reply:
@mechmerlin but that kind of doesn't fit the conventions: https://docs.qmk.fm/#/contributing?id=coding-conventions